docs/conf.py

```diff
@@ -1,39 +1,37 @@
-# -*- coding: utf-8 -*-
-
 import doctest
 import os
 import sys
 from importlib.metadata import version as get_version
 
-sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
We don't enforce quotation styles. We had discussions about this and could not agree. So we just do it manually.